recipe/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pegen" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 0cd306d5b1073542c177035c3caa4526cd10f674a74d6ab40d9a94c57fe1ff33
+
+build:
+  skip: true  # [py<38]
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+    - wheel
```

Comment:
requires `toml`

https://github.com/we-like-parsers/pegen/blob/v0.2.0/pyproject.toml#LL47C57-L47C61

Reply:
and `setuptools_scm`

Reply:
Added!